src/components/messages-view-configurator/MessagesViewConfigurator.tsx

```diff
@@ -50,22 +56,30 @@ const MessageViewConfigurator = ({ sessions }: Props) => {
 		mode === 'body-sort' ? 'active' : null,
 	);
 
+	const offsetTop = document
+		.querySelector('.messages-window-header__configurator-button')
```

Comment:
why would you query an element if you already have a ref. use buttonRef to get getBoundingClientRect values